spec.html

```diff
@@ -42911,7 +42911,7 @@ <h1>%AsyncFromSyncIteratorPrototype%.return ( [ _value_ ] )</h1>
             1. If Type(_result_) is not Object, then
               1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; a newly created *TypeError* object &raquo;).
               1. Return _promiseCapability_.[[Promise]].
-            1. Return ! AsyncFromSyncIteratorContinuation(_result_, _promiseCapability_).
+            1. Return ! AsyncFromSyncIteratorContinuation(_result_, _promiseCapability_, _syncIteratorRecord_, *false*).
```

Comment:
`_syncIteratorRecord_` isn't defined. You presumably need
`1. Let _syncIteratorRecord_ be _O_.[[SyncIteratorRecord]].`
earlier.

Reply:
You're right, bad copy paste!